common/ts/prepare-task.ts

```diff
@@ -118,6 +122,12 @@ function branchName(fullName: string) {
   return fullName;
 }
 
+export function reportPath(): string {
+  return `${tl.getVariable('Agent.TempDirectory')}\\${tl.getVariable(
```

Comment:
Is there a function that will build paths? (i.e. one that would handle using the correct path separators on different platforms?)
If not, it might be better to use "/" instead of  "\" as that is more Linux-friendly and Windows will handle both.

Reply:
Oh yes, totally forgot to use it here. Fixing it right now.